pkg/relayer/session/session.go

```diff
@@ -161,22 +161,31 @@ func (rs *relayerSessionsManager) ensureSessionTree(sessionHeader *sessiontypes.
 	return sessionTree, nil
 }
 
-// mapBlockToSessionsToClaim maps a block to a list of sessions which can be
-// claimed as of that block.
-func (rs *relayerSessionsManager) mapBlockToSessionsToClaim(
+// mapBlockToSessionsToClaimFn returns a new MapFn which maps committed blocks to
+// a list of sessions which can be claimed as of that block.
+func (rs *relayerSessionsManager) mapBlockToSessionsToClaimFn(
 	sessionsToClaimsPublishCh chan<- []relayer.SessionTree,
 ) channel.ForEachFn[client.Block] {
 	return func(_ context.Context, block client.Block) {
 		rs.sessionsTreesMu.Lock()
 		defer rs.sessionsTreesMu.Unlock()
 
-		var lateSessions, onTimeSessions []relayer.SessionTree
+		// onTimeSessions are the sessions that are still within their grace period.
+		// They are on time and will wait for their create claim window to open.
+		// They will be emitted once the late ones, that should no longer wait are emitted.
```

Comment:
Can you update the last line of this comment. Not really clear what you're trying to say.